src/Types.tsx

```diff
@@ -8,7 +14,12 @@ export type PostType = {
 
 export type CommentType = {
   message: string;
-  id: number;
+  id?: number;
   userId: number;
   postId: number;
 };
+
+export type UserType = {
```

Comment:
Usually by convention the types don't have `type` in their name :D Same as naming some variables like this `const postVariable = {something: 1}`